File: Examples/DotNet/CSharp/Worksheet/UnfreezePanes.cs
using Aspose.Cloud;
using System;
namespace Aspose.Cells.Cloud.Examples.Worksheet
{
    class UnfreezePanes
    {
        static void Main()
        {
            string dataDir = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            string input = "sample1.xlsx";
            string output = "ouput.xlsx";
            string sheetName = "Sheet1";

            Common.StorageService.File.UploadFile(dataDir + input, input, storage: Common.STORAGE);

            int row = 1;
            int column = 1;
            int freezedRows = 1;
            int freezedColumns = 1;

            Common.CellsService.Worksheets.UnfreezePanes(input, sheetName, row, column, freezedRows, freezedColumns, Common.FOLDER, storage: Common.STORAGE);

            Common.StorageService.File.DownloadFile(input, dataDir + output, storage: Common.STORAGE);

        }
    }
}

File: Examples/Android/app/src/main/java/com/aspose/cells/cloud/examples/worksheet/UnfreezePanes.java
package com.aspose.cells.cloud.examples.worksheet;


import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;
import java.io.IOException;
import java.io.File;
import com.aspose.cells.cloud.examples.R;
import android.content.Context;


public class UnfreezePanes {

	public static void execute(Context context) throws IOException {
		//ExStart: unfreeze-panes
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			File inputFile = Utils.stream2file("sample1","xlsx", context.getResources().openRawResource(R.raw.sample1));
			File outputFile = Utils.stream2file("sample2","xlsx", context.getResources().openRawResource(R.raw.sample2));

			storageApi.PutCreate("sample1.xlsx", null, Utils.STORAGE, inputFile);

			String sheetName = "Sheet1";
			Integer row = 1;
			Integer column = 1;
			Integer freezedRows = 1;
			Integer freezedColumns = 1;

			cellsApi.DeleteWorksheetFreezePanes("sample1.xlsx", sheetName, row, column, freezedRows,
					freezedColumns, Utils.FOLDER, Utils.STORAGE);

			com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload("sample1.xlsx",
					null, Utils.STORAGE);

			Utils.copyInputStreamToFile(sr.getInputStream(),outputFile);
		}

		catch (Exception e) {
			e.printStackTrace();
		}
		//ExEnd: unfreeze-panes

	}
}